### DISCUSSION TITLE
How to get selected option attribute?

### DISCUSSION THREAD

**Discussion by nimeshgorfad** [sa-si-dev/virtual-select#131] [Q&A] [answered]
How to get selected option attribute?

**Comment by sa-si-dev** [comment, marked as answer]
> How to get selected option attribute?

To get any custom attribute, set it as `customData` property inside the option's object while initializing.

```js
VirtualSelect.init({
  ele: '#sample-select',
  options: [
    { label: 'Options 1', value: '1', customData: { v1: 1 } },
    { label: 'Options 2', value: '2', customData: { v1: 2 } },
  ],
});
```

Use [getSelectedOptions()](https://sa-si-dev.github.io/virtual-select/#/methods?id=getselectedoptions) method to get selected options details

It would return value, label, and customData properties from the options object
